src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let counter = 0;
+
+  const func = (firstNumber) => {
```

Comment:
try to think of a different name for this variable. Functions should describe the action they are doing